src/js/popup.js

```diff
@@ -333,6 +346,26 @@ $(document).ready(function () {
         });
     });
 
+    $('#rate').click(function () {
+        dmUIClick('rate');
+        chrome.runtime.sendMessage({
+            JDTIfunction: 'setRateStatus'
+        }, function (response) {
+            if (response.status === 'success') {
+                // Flag set
+                window.open('https://chrome.google.com/webstore/detail/jira-template-injector/hmhpegjieopgbdmpocdmfkafjgcdmhha/reviews?hl=en', '_blank'); // Open extension user reviews page
```

Comment:
I think we should open the window no matter the out come of saving the boolean. So move this outside of the `if (response.status === 'success') {...`